src/main/java/de/oliver/fancyholograms/commands/hologram/AddLineCMD.java

```diff
@@ -19,6 +19,11 @@ public List<String> tabcompletion(@NotNull CommandSender player, @Nullable Holog
 
     @Override
     public boolean run(@NotNull CommandSender player, @Nullable Hologram hologram, @NotNull String[] args) {
+        if (!(player.hasPermission("fancyholograms.hologram.edit.line.add"))) {
+            MessageHelper.error(player, "You don't have the required permission to add a line to this hologram");
+            return true;
```

Comment:
It would be better to return false. If we return true, the hologram will be updated to the player, but in this case we didn't change anything